update.json

```diff
@@ -7,6 +7,7 @@
 	"pre_update_msg": "§7§m§l--------------------§5§l[§c§lNEU§5§l]§7§m§l--------------------\n\n§7A new pre-release, v§52.0-PRE{pre_version}§7, is now available!\n ",
 	"discord_link": "https://discord.gg/moulberry",
 	"youtube_link": "https://www.youtube.com/channel/UCPh-OKmRSS3IQi9p6YppLcw",
+	"twitch_link": "https://www.twitch.tv/moulberry2",
```

Comment:
this won't work without adding something to the mod

Reply:
https://canary.discord.com/channels/516977525906341928/755816367827058749/871452178601103421